### ISSUE TITLE
The model for blog/models.py is missing an import

### ISSUE THREAD

**Issue by gpaulissen** [wagtail/wagtail#11417]
When running `python manage.py makemigrations` I get:

```
  File "/Users/gpaulissen/dev/t-hive/mysite/blog/models.py", line 13, in BlogPage
    date = models.DateField("Post date")
           ^^^^^^^^^^^^^^^^
```

Versions:
```
python --version
Python 3.11.0

wagtail --version
You are using Wagtail 5.2.2
```

### Pertinent section of the Wagtail docs

[Blog index and posts](https://docs.wagtail.org/en/stable/getting_started/tutorial.html#blog-index-and-posts)

### Details

After adding as first line `from django.db import models` this is solved.

Resulting script:

```
from django.db import models
from wagtail.models import Page
from wagtail.fields import RichTextField
from wagtail.admin.panels import FieldPanel

# add this:
from wagtail.search import index


# Keep the definition of BlogIndexPage model, and add the BlogPage model:

class BlogPage(Page):
    date = models.DateField("Post date")
    intro = models.CharField(max_length=250)
    body = RichTextField(blank=True)

    search_fields = Page.search_fields + [
        index.SearchField('intro'),
        index.SearchField('body'),
    ]

    content_panels = Page.content_panels + [
        FieldPanel('date'),
        FieldPanel('intro'),
        FieldPanel('body'),
    ]
```

**Comment by salvo-polizzi** [comment]
Can I work on this issue?

**Comment by gasman** [comment]
Thanks for your interest @salvo-polizzi! Anyone is free to work on any issue.

We recommend thoroughly reading https://docs.wagtail.org/en/latest/contributing/first_contribution_guide.html before making your first contribution. You may also wish to join [our #new-contributors channel on Slack](https://github.com/wagtail/wagtail/wiki/Slack).

**Comment by gasman** [comment]
Fixed in #11426.
